### PR TITLE
added load_dotenv to read from .env on startup

### PR DESCRIPTION
**Pull request by kenho811-b1**

### CODE REVIEW

File: cancel_all_orders.py [hunk 1]
```diff
@@ -2,9 +2,11 @@
 import os
 from datetime import datetime, timezone
 from hashlib import sha256
-
 import requests
 from eosio_signer import EOSIOKey
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (kenho811-b1) [inline cancel_all_orders.py:9]: read env var from .env on startup

File: cancel_order.py [hunk 2]
```diff
@@ -2,9 +2,11 @@
 import os
 from datetime import datetime, timezone
 from hashlib import sha256
-
 import requests
 from eosio_signer import EOSIOKey
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (kenho811-b1) [inline cancel_order.py:9]: read env var from .env on startup

File: trades_web_socket.py [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 import os
-import ssl
 import threading
 import time
 from collections import deque
-
 import websocket
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (kenho811-b1) [inline trades_web_socket.py:9]: read env var from .env on startup

File: requirements.txt [hunk 4]
```diff
@@ -1,3 +1,4 @@
-git+https://github.com/bullish-exchange/python-signer
+eosio-signer@git+https://github.com/bullish-exchange/python-signer
 websocket-client==1.3.1
 requests==2.25.1
+python-dotenv
```
Comment (kenho811-b1) [inline requirements.txt]: python-dotenv for `load_dotenv()` function

**Review by dexterhuang-blockone — APPROVED** [review]

File: requirements.txt (hunk 4 above)
Comment (dexterhuang-blockone) [reply requirements.txt]: Can u add in the version?

Comment (kenho811-b1) [reply requirements.txt]: let me fix the version for all packages in requirements.txt.

It's a great idea. thanks

File: requirements.txt [hunk 5]
```diff
@@ -1,3 +1,4 @@
-git+https://github.com/bullish-exchange/python-signer
+eosio-signer@git+https://github.com/bullish-exchange/python-signer
 websocket-client==1.3.1
 requests==2.25.1
+python-dotenv==1.0.0
```
Comment (kenho811-b1) [inline requirements.txt:4]: @dexterhuang-blockone version fixed

(I used `pip list|grep python-dotenv` to check the version I am using locally)

**Pull request merged**